README.md

```diff
@@ -61,6 +61,61 @@ taco help
 * Read our [Blog](http://taco.tools/blog/index.html)
 * Have a feature request or find a bug? [Submit an issue](https://github.com/microsoft/taco/issues)
 
+## Development
+
+In order to build the TACO packages, ensure that you have [Git](http://git-scm.com/downloads) and [Node.js](http://nodejs.org/) installed.
+
+Clone a copy of the repo:
+```
+git clone https://github.com/Microsoft/TACO.git
+```
+
+Change to TACO directory:
+```
+cd TACO
+```
+Install dev dependencies
+```
+npm install
+```
+
+### Building TACO
+TACO uses gulp based build system. To build TACO packages, simply run 
+```
+gulp
+```
+Above command will build and install TACO packages under TACO\build\packages\node_modules.
+
+### Running TACO
+Once TACO has been built, it can be run from built location.
+* To run taco-cli run 
+```
+build/packages/node_modules/taco-cli/bin/taco
```

Comment:
Wouldn't it be easier to advice users to `npm link` the package? That way, every time they make a change and compile, they can run `taco` directly without specifying the path.